### PR TITLE
[KOGITO-3467] Script to retrieve list of images

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3467

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: tests/test-apps/clone-repo.sh [hunk 1]
```diff
@@ -17,9 +17,9 @@ if [ $CI ]; then
     cp ${MVN_MODULE}/maven/settings.xml ${HOME}/.m2/settings.xml
     source ${MVN_MODULE}/added/configure-maven.sh
     configure
-fi
 
-cat ${HOME}/.m2/settings.xml
+    cat ${HOME}/.m2/settings.xml
```
Comment (radtriste) [inline tests/test-apps/clone-repo.sh]: this will be corrected by https://github.com/kiegroup/kogito-images/pull/250

**Review by sutaakar — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -4,199 +4,53 @@ BUILD_ENGINE := docker
...
-ifneq ($(findstring rc,$(IMAGE_VERSION)),rc)
-	${BUILD_ENGINE} tag quay.io/kiegroup/kogito-quarkus-ubi8-s2i:${IMAGE_VERSION} quay.io/kiegroup/kogito-quarkus-ubi8-s2i:${SHORTENED_LATEST_VERSION}
-endif
+_build:
+	@for f in $(shell make list); do make build-image image_name=$${f}; done
 
-# build the quay.io/kiegroup/kogito-springboot-ubi8 image
-kogito-springboot-ubi8:
+.PHONY: build-image
+image_name=
```
Comment (spolti) [inline Makefile:26]: the make auto complete will not work in this case, is it a good idea remove it and pass image name for single builds?

Comment (spolti) [reply Makefile:26]: we could maybe, add a option lile, make list-images.

**Review by spolti — CHANGES_REQUESTED** [review]

**Comment by spolti** [comment]
needs rebase.

File: Makefile (hunk 2 above)
Comment (radtriste) [reply Makefile:26]: you mean that one ? 
https://github.com/kiegroup/kogito-images/pull/249/files#diff-b67911656ef5d18c4ae36cb6741b7965R13

Comment (spolti) [reply Makefile:26]: yep, didn't see :)

**Review by spolti — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

**Pull request merged**